tests/configs/test_ablation.template.cfg

```diff
@@ -9,7 +9,7 @@ suffix=.jsonlines
 
 [Tuning]
 grid_search=False
-objective=accuracy
+objective=["accuracy"]
```

Comment:
Why are you changing the templates if the code works fine with a string? There's no reason to.